distribution/docs/src/main/resources/_contents/configuring-from-configuration-files-contents.adoc

```diff
@@ -334,7 +334,7 @@ The following table shows the format to use for each property type supported.
 
 |Lists of Strings
 |`name = [ "value1", "value2", ... ]`
-|`authenticationTypes = [ "/\=SAML\|GUEST", "/admin\=SAML\|basic", "/system\=basic", "/solr\=SAML\|PKI\|basic", "/sources\=SAML\|basic", "/security-config\=SAML\|basic" ]`
+|`complexStringArray = [ "{\"url\"\ \"http://test.sample.com\"\ \"layers\"\ [\"0\"]\ {\"VERSION\"\ \"1.1\|1.2\"\ \"image/png\"}\ \"beta\"\ 1}", "{\"url\"\ \"http://test.sample.com"\ 0.5}", "/solr\=SAML\|PKI\|basic", "/security-config\=SAML\|basic" ]`
```

Comment:
@stustison: This is just an example of what a configuration can look like and is not specific to the Policy Manager or anything else. We were just trying to show the different things that needed to be escaped or not escaped.

To reduce the confusion, we could replace the `solr`, `SAML`, `PKI`, `basic` and `security-config` with non-security names. Would that work?